operator/control-plane/src/aws.rs

```diff
@@ -796,16 +646,80 @@ EOF
                 error!(stderr);
                 return Err(anyhow!("Failed to start logger: {stderr}"));
             }
+        } else {
+            // check if logger is running
+            let (_, stderr) = Self::ssh_exec(sess, "sudo supervisorctl status logger")
+                .context("Failed to start logger")?;
+            if !stderr.is_empty() {
+                // logger is not running
+                return Ok(());
+            }
+
+            // kill the logger
+            let (_, stderr) = Self::ssh_exec(sess, "sudo supervisorctl stop logger")
+                .context("Failed to stop logger")?;
+            if !stderr.is_empty() {
+                error!(stderr);
+                return Err(anyhow!("Failed to stop logger: {stderr}"));
+            }
+        }
+
+        Ok(())
+    }
+
+    // Goal: set up enclave matching enclave.eif, with debug mode if necessary
```

Comment:
what if vcpu and memory are diff?

Reply:
can't be because of the allocator limitation, added a comment

Reply:
tbf, we can just declare that enclaves would be restarted if vcpu and mem change and handle it in the allocator as well :man_shrugging: